### DISCUSSION TITLE
How to change profile picture and banner image on hoppscotch account?

### DISCUSSION THREAD

**Discussion by mrcentimetre** [hoppscotch/hoppscotch#4316] [Q&A] [answered]
![telegram-cloud-photo-size-5-6242546536387034781-y](https://github.com/user-attachments/assets/37211703-0715-4614-a317-0dda9a6fe49e)

I’m currently trying to update my profile picture and banner image on my Hoppscotch account, but I couldn’t find any options to change these elements within the profile settings. As shown in the attached image, the interface doesn’t seem to provide any clickable areas or settings for updating the profile picture or banner. Could someone guide me on how to make these changes, or is this feature currently unavailable?

**Comment by liyasthomas** [comment, marked as answer]
Hoppscotch doesn't support setting a profile picture at the moment.

**Reply by denizyoldasaksu** [reply]
why
